pkg/operator/api/api.go

```diff
@@ -171,13 +171,15 @@ func (c *Cluster) makeDeployment() *extensions.Deployment {
 
 func (c *Cluster) apiContainer() v1.Container {
 
-	command := fmt.Sprintf("/usr/local/bin/rook api --config-dir=%s --port=%d ", k8sutil.DataDir, model.Port)
 	return v1.Container{
-		// TODO: fix "sleep 5".
-		// Without waiting some time, there is highly probable flakes in network setup.
-		Command: []string{"/bin/sh", "-c", fmt.Sprintf("sleep 5; %s", command)},
-		Name:    DeploymentName,
-		Image:   k8sutil.MakeRookImage(c.Version),
+		Command: []string{
+			"/tini", "--", "/usr/local/bin/rook",
```

Comment:
this is already the entrypoint in the Dockerfile. why do you need to repeat it here. Can you not just pass args "api ..."